test/packages/parallel/oracle/data_stream/memory/_dev/test/system/test-memory-config.yml

```diff
@@ -0,0 +1,16 @@
+vars:
+  hosts:
+    - "oracle://sys:Oradoc_db1@elastic-package-service-oracle-1:1521/ORCLCDB.localdomain?sysdba=1"
```

Comment:
This is required to be changed since the container name changes from `oracle` to `elastic-package-service-oracle-1`.